channels/drdynvc/client/drdynvc_main.c

```diff
@@ -1510,9 +1511,12 @@ static UINT drdynvc_virtual_channel_event_connected(drdynvcPlugin* drdynvc, LPVO
 		return status;
 	}
 
-	settings = (rdpSettings*)drdynvc->channelEntryPoints.pExtendedData;
+	WINPR_UNUSED(drdynvc->rdpcontext);
+	settings = drdynvc->rdpcontext->settings;
```

Comment:
Copy past error, should it not be WINPR_ASSERT?

Reply:
true.